pjsip/src/pjsua-lib/pjsua_acc.c

```diff
@@ -3519,7 +3519,7 @@ pj_status_t pjsua_acc_get_uac_addr(pjsua_acc_id acc_id,
     {
         int i;
 
-        for (i = 0; i < (int)sizeof(pjsua_var.tpdata); i++) {
+        for (i = 0; i < PJ_ARRAY_SIZE(pjsua_var.tpdata); i++) {
```

Comment:
To supress build warning
```
../src/pjsua-lib/pjsua_acc.c:3522:23 warning: 
comparison of integers of different signs: 'int' and 'unsigned long' [-Wsign-compare]
           for (i = 0; i < PJ_ARRAY_SIZE(pjsua_var.tpdata); i++) {
                     ~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

should change to `(int)`PJ_ARRAY_SIZE(pjsua_var.tpdata)